### PR TITLE
ember-data najax deprecation

### PR DESCRIPTION
**Pull request by snewcomer**
najax deprecation

https://github.com/emberjs/data/pull/7230/files

### CODE REVIEW

**Comment by locks** [comment]
Let us know when this is ready for review!

**Comment by snewcomer** [comment]
It is ready for review!

**Comment by igorT** [comment]
LGTM but @rwjblue should also probably take a look

File: content/ember-data/v3/najax.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+id: ember-data:najax-fallback
+title: Deprecate najax request
+until: '4.0.0'
+since: '3.22.0'
+---
+#### Deprecates the adapter `najax`
```
Comment (igorT) [inline content/ember-data/v3/najax.md:7]: I wonder if this could be phrased slightly differently, this sounds to me like an adapter class being deprecated

**Review by locks — APPROVED** [review]
Thanks for writing out the deprecation guide! I will be merging once the checks are green. If any improvements pop up, they will also be welcome :)

**Pull request merged**